### PR TITLE
[UP] Suggestion to fix issue 79 and 71

### PR DESCRIPTION
**Pull request by gsilvapt**
Suggestion to fix issue [71](https://github.com/okfn/datapatterns/issues/71) and [79](https://github.com/okfn/datapatterns/issues/79)

Right after push, got this email from [GitHub about custom pages](http://prntscr.com/azfa9b).

Never seen that so I thought it would be better to show you guys to make sure there is nothing wrong.

### CODE REVIEW

File: _layouts/pattern.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+layout: default
+type: pattern
+
```
Comment (rufuspollock) [inline _layouts/pattern.html]: Why the blank line?

**Comment by rufuspollock** [comment]
Generally looks good.

Couple of items:
- glossary is at `/glossary/glossary.md` surely this should be `/glossary/index.md`?
- You have this in the commit which is surely accidental:
  
  ```
  .Rhistory
  ```
  
  To correct you need not only to remove but to "squash" the commits because we do not want this in the commit history if possible.

**Comment by gsilvapt** [comment]
I have no idea how that R temp file went over there. I haven't used R in a while!?

Anyway, I **think** I did everything correctly this time. Apologies for the lack of integrity there. Please let me know if I wasn't able to do it. I tried via terminal and it was just a huge mess but according to GitHub, it is worked.

Thank you for the feedback

**Comment by rufuspollock** [comment]
Great!. Looks good.

**Pull request merged**